generator/src/main/java/org/corfudb/generator/operations/RemoveOperation.java

```diff
@@ -2,23 +2,34 @@
 
 import java.util.UUID;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.UUID'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)